apigentools/cli.py

```diff
@@ -392,6 +392,10 @@ def get_cli_parser():
         action="store_true",
     )
 
+    sp.add_parser(
+        "list-languages", help="Provide a list of the supported langauges for this spec"
```

Comment:
Can we do either options `--list-languages` and `--list-versions` or command `list (languages|versions)`?

Reply:
Or maybe just `--list` that would list language-version combinations and would have `--versions-only` and `--languages-only` mutually exclusive switches?

Reply:
Hmm, if they are options I'm not sure which command it should be tied to. 
I can make `list-languages` take in like a `--include-versions` and show each version alongside it. WDYT?

Reply:
I can go that route ^. I may call it something like `apigentools spec-info` though 🙂 If that approach works for you guys I'll start that route

Reply:
I think this is a good route to take, however I'm wondering whether `spec-info` is the right thing to call it. It almost feel like `spec-info` is too broad. However, I don't have a better proposal, so I'll leave this point open to discussion :)

Reply:
I left it as list-languages, but I've addressed the other review from this thread + used the utility method we have in the base Command class. So if we're ok with `list-languages` I think this is good to go.